### DISCUSSION TITLE
Remove one custom dns resolution

### DISCUSSION THREAD

**Discussion by rojagtap** [burningwave/tools#4] [Q&A] [answered]
I want to add custom dns resolution for a few hosts. I understand that this can be achieved via `HostResolutionRequestInterceptor.INSTANCE.install`. However, I would like to understand if there is a way to remove one entry from the dns resolvers

**Comment by Roberto-Gentili** [comment, marked as answer]
No, you can't. But you can uninstall all previous hostresolvers installed through the method `HostResolutionRequestInterceptor.INSTANCE.uninstall` and reinstall a new collection again

**Reply by rojagtap** [reply]
Okay, understood. Also how expensive is this: uninstall + re install?

**Reply by Roberto-Gentili** [reply]
It shouldn't be a very heavy operation
